src/platform/mhv/secondary-nav/components/MhvSecondaryNavItem.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * A secondary nav item.
+ * @param iconClass the classname(s) for a font awesome icon
+ * @param href the link for the navigation item
+ * @param title the title for the navigation item
+ * @param abbreviation the abbreviation for the navigation item shown instead of the title when the width is less than 400px
+ * @param isActive true if the nav item is to be shown as active
+ * @returns a secondary nav item
+ */
+const MhvSecondaryNavItem = ({
+  iconClass,
+  href,
+  title,
+  abbreviation,
+  isActive = false,
+  isHeader = false,
+}) => {
+  const key = title.toLowerCase().replaceAll(' ', '_');
+  const mobileTitle = abbreviation ? (
+    <abbr title={title}>{abbreviation}</abbr>
+  ) : (
+    title
+  );
+
+  const activeItemClasses = isActive
+    ? 'mhv-u-sec-nav-active-style vads-u-font-weight--bold'
+    : '';
+  const itemTypeClasses = isHeader
+    ? 'mhv-u-sec-nav-header-style vads-u-font-size--lg'
+    : 'mhv-u-sec-nav-item-style';
```

Comment:
FYI: The [classnames](https://www.npmjs.com/package/classnames) node module provides a convenient way for dealing with conditional class names.

Reply:
Oh! Me like!